static/src/stylesheets/module/_weather.scss

```diff
@@ -134,128 +274,92 @@ $weather-large-size: 60px;
-    position: relative;
-    @include transition(opacity .2s ease-in); // Needs Fixing
-
-    @include mq($until: leftCol) {
-        @include box-sizing(border-box);
-        width: gs-span(1) + $gs-gutter;
-        float: left;
-        padding-right: $weather-small-size;
-        text-align: right;
+    clear: both;
+    margin: 0;
+    padding: ($gs-baseline / 3) 0 ($gs-baseline / 2);
+    list-style: none;
+    line-height: 1.2;
+    display: inline-block;
 
-        .weather__icon {
-            position: absolute;
-            top: 0;
-        }
+    @include mq($until: mobileLandscape) {
+        display: table;
+        width: 100%;
     }
 
-    @include mq(tablet, leftCol) {
-        padding-right: $weather-small-size + $gs-gutter / 2;
+    @include mq(mobileLandscape, leftCol) {
+        float: right;
+    }
 
-        &:after {
-            content: '';
-            position: absolute;
-            top: 3px;
-            right: $gs-gutter / 4;
-            height: $gs-baseline + 1px;
-            border-right: 1px dotted colour(neutral-3);
-        }
+    @include mq(leftCol) {
+        display: table;
+        width: 100%;
 
-        .weather__icon {
-            right: $gs-gutter / 2;
+        &.weather__forecasts--mobile {
+            display: none;
         }
     }
 
-    .has-page-skin & {
-        @include mq(wide) {
-            @include box-sizing(border-box);
-            padding-right: $weather-small-size + $gs-gutter / 2;
-            width: gs-span(1) + $gs-gutter;
-            float: left;
-            text-align: right;
+    .weather__forecast {
+        position: relative;
+        display: table-cell;
+        border-left: 1px dotted colour(neutral-3);
+        padding-left: $gs-gutter / 4;
+        padding-right: floor($gs-gutter / 1.333);
 
-            .weather__icon {
-                position: absolute;
-                top: 0;
-            }
+        @include mq(mobileLandscape) {
+            min-width: gs-span(1) - ($gs-gutter / 4);
+        }
 
-            &:after {
-                content: '';
-                position: absolute;
-                top: 3px;
-                right: $gs-gutter / 4;
-                height: $gs-baseline + 1px;
-                border-right: 1px dotted colour(neutral-3);
+        @include mq(leftCol) {
+            padding-right: 0;
+        }
+
+        &:first-of-type {
+            @include mq($until: mobileLandscape) {
+                padding-left: 0;
+                border-left: 0;
             }
 
-            .weather__icon {
-                right: $gs-gutter / 2;
+            @include mq(leftCol) {
+                padding-left: 0;
+                border-left: 0;
             }
         }
-    }
-}
 
-.weather__desc {
-    @include fs-headline(2);
-    margin-right: 5px;
+        @include mq(leftCol, wide) {
+            min-width: 33%;
 
-    @include mq(leftCol) {
-        position: absolute;
-        bottom: 2px;
-    }
+            &--3 {
```

Comment:
I don't know what our general guidelines are for this. But I for one am not down with these new hipster™ Sass BEM helpers. They make grepping the codebase much harder.